js/controller.js

```diff
@@ -275,6 +275,7 @@ OO.plugin("Html5Skin", function (OO, _, $, W) {
 
     onEmbedCodeChanged: function(event, embedCode, options) {
       this.state.videoQualityOptions.availableBitrates = null;
+      this.state.closedCaptionOptions.availableLanguages = null;
```

Comment:
Looks like we should null everything we store from the previous video here. For example, discoveryData should not carryover as well.
@brianfwong, could you set the discoveryData to null as well here?